src/API/skyblock/getAuction.js

```diff
@@ -0,0 +1,19 @@
+const Errors = require('../../Errors');
+const toUuid = require('../../utils/toUuid');
+module.exports = async function (type, query, includeItemBytes = false) {
+  if (!query) throw new Error(Errors.NO_NICKNAME_UUID);
+  const Auction = require('../../structures/SkyBlock/Auctions/Auction');
+  let parameters = '';
+  if (type === 'PROFILE') {
+    parameters = `profile=${query}`;
+  } else if (type === 'PLAYER') {
+    query = await toUuid(query);
+    parameters = `player=${query}`;
+  } else if (type === 'AUCTION') {
+    parameters = `uuid=${query}`;
+  } else throw new Error(Errors.BAD_AUCTION_FILTER);
+  const res = await this._makeRequest(`/skyblock/auction?${parameters}`);
```

Comment:
imma kms if i wrote this

```suggestion
  let filter = '';
  if (type === 'PROFILE') {
    filter = 'profile';
  } else if (type === 'PLAYER') {
    query = await toUuid(query);
    filter = 'player';
  } else if (type === 'AUCTION') {
    filter = 'uuid';
  } else throw new Error(Errors.BAD_AUCTION_FILTER);
  const res = await this._makeRequest(`/skyblock/auction?${filter}=${query}`);
```